src/assignment/views/AssignmentResponseEdit/tabs/AssignmentResponseSearchTab.tsx

```diff
@@ -45,6 +46,12 @@ interface AssignmentResponseSearchTabProps {
 	appendBlockToPupilCollection: (block: Avo.Core.BlockItemBase) => void; // Appends a block to the end of the list of blocks of the current (unsaved) pupil collection
 }
 
+// TODO: avoid hard-coded strings?
+const ElementaryEducationLevels = ['Lager onderwijs', 'Kleuteronderwijs'].flatMap((level) => [
```

Comment:
shouldn't the filter work with labels and ids
labels being 
```
[
	'Lager onderwijs', 
	'Kleuteronderwijs'
]
```
and ids being 
```
[
	'https://w3id.org/onderwijs-vlaanderen/id/structuur/lager-onderwijs', 
	'https://w3id.org/onderwijs-vlaanderen/id/structuur/kleuteronderwijs'
]
```

Reply:
It should, but it doesn't 😅
<img width="1512" alt="Screenshot 2024-05-16 at 11 37 14" src="https://github.com/viaacode/avo2-client/assets/90180417/a28c1e94-5cea-4251-9ecf-85f5e2d012db">